src/tests/ftest/io/nvme_io.py

```diff
@@ -24,147 +24,140 @@
 from __future__ import print_function
 
 import os
-import sys
-import json
-import traceback
-import uuid
-import re
 import avocado
-from apricot import TestWithServers
 
+from apricot import TestWithServers
+from daos_api import DaosPool, DaosApiError
+from general_utils import DaosTestError
+from ior_utils import IorCommand, IorFailed
 import write_host_file
-import ior_utils
 
-from general_utils import DaosTestError
-from daos_api import DaosPool, DaosApiError
 
 class NvmeIo(TestWithServers):
```

Comment:
Should this be subclassing `IorTestBase`?

Reply:
Yes